### DISCUSSION TITLE
Overset Mask in MLCellABecLap

### DISCUSSION THREAD

**Discussion by ankithadas** [AMReX-Codes/amrex#3511] [Q&A] [answered]
Hey AMReX team,

I quite couldn't figure out the purpose of m_overset_mask in MLCellABecLap.H. I can see that it is used a lot in MLABecLaplacian class and other derived classes. Any explanation would be much appreciated 

Kind regards,
Ankith

**Comment by WeiqunZhang** [comment, marked as answer]
google "overset method"

**Reply by WeiqunZhang** [reply]
Not that I do not want to explain. I don't know anything insightful about it. I just implemented a feature request. Basically some points are taken of the system and do not count as unknowns.
